scripts/helpers/isNetwork.ts

```diff
@@ -18,4 +18,9 @@ export function isAvalancheFuji(hardhat: HardhatRuntimeEnvironment) {
 export function isOptimismKovan(hardhat: HardhatRuntimeEnvironment) {
   const network = hardhat.network.name
   return /optimism/.test(network) || /kovan/.test(network) || /optimismkovan/.test(network)
+}
+
+export function isArbitrumRinkeby(hardhat: HardhatRuntimeEnvironment) {
+  const network = hardhat.network.name
+  return /arbitrum/.test(network) || /rinkeby/.test(network) || /arbitrumrinkeby/.test(network)
```

Comment:
We should remove `/rinkeby/` cause if the name of the network passed is rinkeby, it will return true.
```suggestion
  return /arbitrum/.test(network) || /arbitrumrinkeby/.test(network)
```